modules/ROOT/pages/procedures.adoc

```diff
@@ -263,7 +263,7 @@ For more information, see the link:{neo4j-docs-base-uri}/cdc/{page-version}/[CDC
 | *Mode* 3+| READ
 |===
 
-[role=label--enterprise-edition label--new-5.17 label--beta label--admin-only]
+[role=label--enterprise-edition  label--beta label--admin-only]
```

Comment:
```suggestion
[role=label--enterprise-edition  label--beta label--GA-5.23 label--admin-only]
```

Reply:
or to remove the `beta` label and not to use the `GA` label?

Reply:
I think that we should remove all mentions of 5.x, including GAs, etc. If the feature is no longer beta, we should also remove beta labels.